.github/workflows/submit-beta.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       - name: Create .env file
         run: |
           echo "PLASMO_PUBLIC_TRANSAK_API_KEY=${{ secrets.TRANSAK_API_KEY }}" >> .env
-          echo "PLASMO_PUBLIC_APP_TYPE=${{ secrets.APP_TYPE }}" >> .env
```

Comment:
I've removed these ENV variables as now they are only needed for the Embedded (Vite) build. Anything else just falls back to the BE build.